package.json

```diff
@@ -10,12 +10,12 @@
     "@babel/preset-env": "^7.11.5",
     "@babel/runtime": "^7.11.2",
     "acorn": "^7.4.0",
-    "autoprefixer": "^9.8.6",
+    "autoprefixer": "^10.0.1",
     "babel-loader": "^8.1.0",
     "canvas": "^2.6.1",
     "core-js": "^3.6.5",
     "cross-env": "^7.0.2",
-    "es-module-shims": "^0.4.7",
+    "es-module-shims": "^0.6.0",
```

Comment:
Given that this dependency is only used for development, and isn't covered by the test-suite, please confirm that the following things work locally (after running `gulp server` of course):
 - The viewer itself, and the default tracemonkey document, loads successfully when opened with http://localhost:8888/web/viewer.html 
 - The unit-tests completes successfully (with the possible exception of one Worker-related test failing), when run with http://localhost:8888/test/unit/unit_test.html

Reply:
I have verified that this indeed works as described. Thank you for reviewing this, and the other patch!